master/buildbot/config.py

```diff
@@ -449,7 +451,7 @@ def load_caches(self, filename, config_dict):
             if not isinstance(caches, dict):
                 error("c['caches'] must be a dictionary")
             else:
-                valPairs = caches.items()
+                valPairs = list(iteritems(caches))
```

Comment:
Right but valPairs is not used except to iterate (in which case im not sure having it as a local var really helps readability) .. or is it used and I'm missing it?